doc/HttpLuaModule.wiki

```diff
@@ -1277,6 +1277,36 @@ This directive was first introduced in the <code>v0.9.5</code> release.
 
 This hook no longer runs in the cache manager and cache loader processes since the <code>v0.10.12</code> release.
 
+== exit_worker_by_lua_block ==
+
+'''syntax:''' ''exit_worker_by_lua_block { lua-script }''
+
+'''context:''' ''http''
+
+'''phase:''' ''exiting-worker''
+
+Runs the specified Lua code upon every Nginx worker process's exiting when the master process is enabled. When the master process is disabled, this hook will before run Nginx process exiting.
+
+This hook is often used to release resources when [[#init_worker_by_lua|init_worker_by_lua*]] create resources, to prevent the worker from exiting abnormally.
```

Comment:
This can be clarified and can use better grammar as well:

```
This hook is ogten used to release resources allocated by each worker (e.g. resources allocated by init_worker_by_lua), or to prevent workers from exiting abnormally.  
```